tests/autoagora_agents/test_algorithm.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2022-, Semiotic AI, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import torch
+
+from autoagora_agents import algorithm
+
+from ..fixture import *
+
+
+def test_predetermined(predeterminedconfig):
```

Comment:
where `predeterminedconfig` is coming... is it like a fixture?

Reply:
Yup. All the fixtures are in one place. https://github.com/semiotic-ai/autoagora-agents/blob/7ebc3283b9df6d4455d0ee807ae9ab1de7b18a59/tests/fixture.py